src/main.ts

```diff
@@ -1,51 +1,25 @@
-import Vue from 'vue';
-import VueI18n from 'vue-i18n';
-import i18n from './i18n';
+import { App, defineCustomElement } from 'vue';
 import PsAccounts from '@/components/PsAccounts.vue';
-import AccountPanel from '@/components/panel/AccountPanel.vue';
-import {isOnboardingCompleted} from '@/lib/Helpers';
+import { isOnboardingCompleted } from '@/lib/Helpers';
 
-const version = process.env.VUE_APP_VERSION;
-
-const Components: Record<string, Vue.VueConstructor> = {
-  PsAccounts,
-  AccountPanel,
+const installGlobalComponents = {
+  install (app: App) {
+    app.component('PsAccounts', PsAccounts);
+  }
 };
 
-const Plugin = {
-  install(vue: typeof Vue) {
-    vue.use(VueI18n);
+const PsAccountsWebComponent = defineCustomElement(PsAccounts);
 
-    Object.keys(Components).forEach((name) => {
-      vue.component(name, Components[name]);
-    });
-  },
+const init = () => {
+  customElements.define('prestashop-accounts', PsAccountsWebComponent);
```

Comment:
No worry, not a new review @aAmorim27 :sweat_smile:. Just a thought, why not define Web Component for the page without having to call the init function?

Reply:
I had part of an answer in the documentation here: https://vuejs.org/guide/extras/web-components.html#building-custom-elements-with-vue

It is more for flexibility. In our context, maybe we can define it without the init function. That can be done later. Once again, just a thought.

Reply:
It's to avoid breaking the component implementation as every module that integrates the component currently uses the init function pour utiliser le composant
The Web component is also exported, the user can still use the customElements.define function to add the web component by himself in his module